container-core/src/main/java/com/yahoo/jdisc/http/server/jetty/ConnectorFactory.java

```diff
@@ -75,7 +75,7 @@ public ServerConnector createConnector(final Metric metric, final Server server,
         connector.setName(connectorConfig.name());
         connector.setAcceptQueueSize(connectorConfig.acceptQueueSize());
         connector.setReuseAddress(connectorConfig.reuseAddress());
-        connector.setIdleTimeout(toMillis(connector.getIdleTimeout()));
+        connector.setIdleTimeout(toMillis(connectorConfig.idleTimeout()));
```

Comment:
Perhaps a unit test to verify that config is actually propagated correctly ?

Reply:
https://github.com/vespa-engine/vespa/pull/18182